src/lib/pci_virtio_9p.c

```diff
@@ -21,7 +21,7 @@
 #define VIRTIO_9P_MOUNT_TAG 1
 
 static int pci_vt9p_debug = 0;
-#define DPRINTF(params) if (pci_vt9p_debug) printf params
+#define DPRINTF(params) if (pci_vt9p_debug) log_info params
```

Comment:
there shouldn't be any need to change this debug code. this won't be enabled for release builds and there should be no need to redirect to a logger.